website/package.json

```diff
@@ -12,12 +12,13 @@
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
-    "typecheck": "tsc"
+    "tsc": "tsc"
   },
   "dependencies": {
     "@algolia/client-search": "^4.14.3",
     "@docusaurus/core": "2.2.0",
     "@docusaurus/preset-classic": "2.2.0",
+    "@docusaurus/theme-classic": "^2.4.0",
```

Comment:
```suggestion
```
from the doc: https://docusaurus.io/docs/api/themes/@docusaurus/theme-classic
> If you have installed @docusaurus/preset-classic, you don't need to install it as a dependency.

Reply:
After updating eslint & typescript there is an error if we don't add this.